src/LibraryManager/Providers/Unpkg/UnpkgProvider.cs

```diff
@@ -25,7 +25,7 @@ public UnpkgProvider(IHostInteraction hostInteraction, CacheService cacheService
         public override ILibraryCatalog GetCatalog()
         {
             // TODO: sort out the WebRequestHandler dependency
```

Comment:
This comment seems like it no longer applies, since the `WebRequestHandler.Instance` parameter was removed, no?